### PR TITLE
i18n(pt): Translate `resources/showcase.mdx`

### PR DESCRIPTION
**Pull request by viniciusdeliz**
- Create resources folder

- Create showcase.mdx file inside /resources

#### Description

- Translate showcase.mdx file from documentation

### CODE REVIEW

File: docs/src/content/docs/pt-br/resources/showcase.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Vitrine Starlight
+description: Descubra sites construídos com Starlight!
+sidebar:
+  label: Vitrine Starlight
```
Comment (delucis) [inline docs/src/content/docs/pt-br/resources/showcase.mdx:5]: Noting that in the English the sidebar label and title are not the same: the page title is “Starlight Showcase” but the sidebar label is “Site Showcase”. I guess the same might apply here?

Comment (viniciusdeliz) [reply docs/src/content/docs/pt-br/resources/showcase.mdx]: Great catch, thank you for pointing that out, @delucis. I've just updated the translation with a new commit.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks

**Review by delucis — APPROVED** [review]
Great! Thank you for contributing @viniciusdeliz ✨

**Pull request merged**